pallets/liquidity-pools/src/lib.rs

```diff
@@ -689,6 +692,22 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		#[pallet::weight(10_000)]
+		#[pallet::call_index(10)]
+		pub fn schedule_upgrade(
+			origin: OriginFor<T>,
+			evm_chain_id: EVMChainId,
```

Comment:
@mustermeiszer would you be able to show me how to solve this `cannot find type `EVMChainId` in this scope` error here? I'm not exactly sure how to get this type in scope.